src/main/jbake/content/docs/components/metrics/index.adoc

```diff
@@ -4,22 +4,874 @@ Thomas Segismont
 :icons: font
 :jbake-type: page
 :jbake-status: published
+:toc: macro
+:toc-title:
 
-Hawkular Metrics is the metric data store part of the http://www.hawkular.org/[Hawkular] organization.
-It is currently used as the metric storage engine for Hawkular but it can also be used independently. The project relies on https://cassandra.apache.org/[Apache Cassandra] as a backend.
+toc::[]
 
+== Introduction
+Hawkular Metrics is a scalable, asynchronous, multi tenant, long term metrics storage engine that uses
+ link:http://cassandra.apache.org[Cassandra] as the data store and REST as the primary interface. It
 
-Three types of metrics are currently supported:
+*Scalability* +
+Hawkular Metrics is all about scalability. You can run a single instance backed by a single Cassandra node. You can
+also scale out Cassandra to multiple nodes to handle increasing loads. The Hawkular Metrics server employs a stateless
+architecture, which makes it easy to scale out as well.
 
- * Availability: metric that represent availability: up or down
- * Counter: numeric value that always goes up; example: the total number of visitors of a web page
-   ** Rates: rate of change can be retrieve for counter rates
- * Gauge: numeric value that can go up or down; example: the response time on an application
-   ** Aggregate buckets (min/max/average on some intervals) can be obtained for gauges when individual datapoints are not needed
+image::metrics_scalability.png[]
```

Comment:
It was not working for me locally. I wasn't sure if maybe it was due to some quirk with the local, dev server,so I decided to commit/push the change to see if the image rendered. I am not sure why the image fails to render. Maybe @Jiri-Kremser has an idea.